src/routes/roomTypes.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ao
+ * Date: 2018-11-12
+ * Time: 12:34 AM
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+// Get All room types for a hotel
+$app->get('/api/hotels/{id}/room-types', function (Request $request, Response $response, array $args) {
+
+    $hotel_id = $args['id'];
+
+    $sql = "SELECT * FROM room_type where hotel_id = '$hotel_id' ";
+    try {
+        // Get DB Object
+        $db = new db();
+        // Connect
+        $db = $db->connect();
+        $stmt = $db->query($sql);
+        $customers = $stmt->fetchAll(PDO::FETCH_OBJ);
+        $db = null;
+
+        return $response->write(json_encode($customers));
+    } catch (PDOException $e) {
+
+        $db = null;
+
+        $response->write($e);
+        return $response->withStatus(500);
+    }
+});
+
+// create a room type to a hotel
+$app->post('/api/hotels/{id}/room-types', function (Request $request, Response $response, array $args) {
+
+    $hotel_id = $args['id'];
+
+    $sql = "INSERT INTO `room_type` (`id`, `type_name`, `occupancy`, `description`, `price`, `available_slots`, `hotel_id`, `total_slots`)" .
```

Comment:
you dont need the quote on `room_type`

Reply:
we need to talk about this `available_slots` column, we talked about this being useless right? How would we differentiate the number of available slots between 2 years in the future and tmr?